suite-native/module-send/src/components/CoinControl/SendUtxoScreenHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import { useAlert } from '@suite-native/alerts';
+import { IconButton, Text } from '@suite-native/atoms';
+import { Translation, useTranslate } from '@suite-native/intl';
+import { GoBackIcon, ScreenHeader } from '@suite-native/navigation';
+
+import { useUtxoSelection } from '../../hooks/useUxtoSelection';
+
+type CoinControlScreenHeaderProps = {
+    onDelete?: () => void;
+};
+
+export const SendUtxoScreenHeader = ({ onDelete }: CoinControlScreenHeaderProps) => {
+    const { selectedUtxos, setSelectedUtxos } = useUtxoSelection();
+    const { translate } = useTranslate();
+    const { showAlert } = useAlert();
+
+    const handleDelete = () => {
+        showAlert({
+            title: translate('moduleSend.coinControl.disable.title'),
+            description: translate('moduleSend.coinControl.disable.description'),
+            primaryButtonTitle: translate('moduleSend.coinControl.disable.primaryButton'),
+            secondaryButtonTitle: translate('moduleSend.coinControl.disable.secondaryButton'),
+            onPressPrimaryButton: () => {
+                setSelectedUtxos([]);
+                onDelete?.();
+            },
+            primaryButtonVariant: 'redBold',
+            secondaryButtonVariant: 'redElevation0',
+        });
+    };
+
+    return (
+        <ScreenHeader
+            content={
+                <Text>
+                    <Translation id="moduleSend.coinControl.title" />
+                </Text>
+            }
+            leftIcon={<GoBackIcon closeActionType="close" />}
```

Comment:
ScreenHeader already has a prop handling this.
```suggestion
    closeActionType="close"
```

Reply:
ca79ab1df38447a6c4cfe999a927ae41a793df59